classes/UpgradeTools/CoreUpgrader/CoreUpgrader.php

```diff
@@ -377,20 +379,20 @@ protected function applySqlParams(array $sqlFiles): array
      * @param string $upgrade_file File in which the request is stored (for logs)
      * @param string $query
      */
-    protected function runQuery(string $upgrade_file, string $query)
+    protected function runQuery(string $upgrade_file, string $query): void
     {
         $query = trim($query);
         if (empty($query)) {
             return;
         }
         // If php code have to be executed
         if (strpos($query, '/* PHP:') !== false) {
-            return $this->runPhpQuery($upgrade_file, $query);
+            $this->runPhpQuery($upgrade_file, $query);
```

Comment:
An early return was probably expected here:

```suggestion
            $this->runPhpQuery($upgrade_file, $query);
            return;
```